config/core/test-configs.yaml

```diff
@@ -2281,6 +2281,18 @@ test_configs:
   - device_type: qcom-qdf2400
     test_plans:
       - baseline
+      - kselftest-filesystems
```

Comment:
Failing to boot:
https://staging.kernelci.org/test/plan/id/61e2ba568d679f1af9217830/

Looks like it may be an issue with the kernels build with the kselftest fragment.